lib/mayaUsd/render/MaterialXGenOgsXml/libraries/1.38.7/mx_srgb_texture_to_aces_2065_1_color3.glsl

```diff
@@ -0,0 +1,11 @@
+#include "libraries/stdlib/genglsl/lib/mx_transform_color.glsl"
```

Comment:
All these new files in the 1.37.7 folder are copies of files modified for 1.38.8 before changes.